pkg/tunneldriver/driver.go

```diff
@@ -52,10 +53,29 @@ type TunnelDriverOpts struct {
 	Region     string
 }
 
+type TunnelDriverComments struct {
+	Gateway string `json:"gateway,omitempty"`
+}
+
 // New creates and initializes a new TunnelDriver
-func New(logger logr.Logger, opts TunnelDriverOpts) (*TunnelDriver, error) {
+func New(logger logr.Logger, opts TunnelDriverOpts, tunnelComments []TunnelDriverComments) (*TunnelDriver, error) {
```

Comment:
I think we only really need one of these. Anything else we add to it will be a field on the struct rather than another entry in this slice.

Reply:
un-sliced here: https://github.com/ngrok/kubernetes-ingress-controller/pull/357/commits/e26e0a3a8cb2a78653dd00978939154a76f0fb07